script/build_apm_docs.sh

```diff
@@ -17,16 +17,12 @@ else
     git -C $docs_dir pull https://github.com/elastic/docs.git
 fi
 
-index_list="$(find ${GOPATH%%:*}/src/$path -name 'index.asciidoc')"
-for index in $index_list
-do
-  echo "Building docs for ${name}..."
-  echo "Index document: ${index}"
-  index_path=$(basename $(dirname $index))
-  echo "Index path: $index_path"
+echo "Building docs for ${name}..."
+echo "Index document: ${path}"
+index_path=$(basename $(dirname $path))
+echo "Index path: $index_path"
 
-  dest_dir="$html_dir/${name}/${index_path}"
-  mkdir -p "$dest_dir"
-  params="--chunk=1"
-  $docs_dir/build_docs --direct_html $params --doc "$index" --out "$dest_dir"
-done
+dest_dir="$html_dir/${name}/${index_path}"
+mkdir -p "$dest_dir"
+params="--chunk=1"
+$docs_dir/build_docs --direct_html $params --doc "$path" --out "$dest_dir"
```

Comment:
`--direct_html` is on by default now. Might as well clean that up while we're here.

```suggestion
$docs_dir/build_docs $params --doc "$path" --out "$dest_dir"
```